src/app/space-settings/areas/create-area-dialog/create-area-dialog.component.ts

```diff
@@ -4,6 +4,10 @@ import { Modal } from 'ngx-modal';
 import { Context, AreaService, Area, AreaAttributes } from 'ngx-fabric8-wit';
 import { ContextService } from '../../../shared/context.service';
 
+interface Errors {
+  unique: boolean
```

Comment:
or you can add the Error interface to https://github.com/fabric8-ui/fabric8-ui/tree/master/src/app/error